apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -1151,8 +967,6 @@ export class ApplicationController {
     // delete the entry in Payment table to prevent FK error
     await this.paymentService.delete(existingApplication.id, user)
 
-    await this.fileService.deleteAttachmentsForApplication(existingApplication)
```

Comment:
Should this be removed? Isn't this for pruning all attachments e.g. file-uploads?

Reply:
Great catch, thanks. That was meant to stay.